### ISSUE TITLE
Tables from Summary values

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#3393]
Currently the frequency tables and summary tables (and column summaries) can produce the summary statistics in the "long" format.
Please could we consider having the 2-way table from this format.  The menu item is already there, namely Describe > General > Use Summaries.

**Comment by dannyparsons** [comment]
Yes, shouldn't be difficult to have a method to convert this to an HTML

**Comment by rdstern** [comment]
There are three other points here:
1) If there is just one column factor then this should be possible also with an unstack and then display in the output window.  
2) This might be a useful option anyway, partly because it would be quick and cheerful.  (And sometimes the HTML tables take a long time to process if the table is large and are also given ina browser until that bug is fixed in R.
3) If the counts are given, then (perhaps later) it should be possible to give some margins, and all the common ones, like sums and means and percents.  (Others will necessitate going back to the primary data, so they wouldn't be allowed.

**Comment by dannyparsons** [comment]
The simplest option is to use the summary table method and do a "summary" of the data like we do for display daily in climatic. This then wouldn't require a new method and would be quick to do. If we're happy with this I can specify the task for someone.
